patches/minecraft/net/minecraft/client/audio/SoundHandler.java.patch

```diff
@@ -1,14 +1,36 @@
 --- ../src-base/minecraft/net/minecraft/client/audio/SoundHandler.java
 +++ ../src-work/minecraft/net/minecraft/client/audio/SoundHandler.java
-@@ -67,6 +67,7 @@
+@@ -33,7 +33,7 @@
+ import org.apache.logging.log4j.Logger;
```

Comment:
Import in patch file.

Reply:
I mistaked it for that too -- it's just patch context from the below implements change.

Reply:
Ah yeah, sorry about that.